### PR TITLE
fix form completion report filter

### PR DESCRIPTION
**Pull request by snopoke**
http://manage.dimagi.com/default.asp?185285
@czue or anyone

### CODE REVIEW

**Comment by snopoke** [comment]
:white_check_mark:

File: corehq/apps/reports/standard/monitoring.py [hunk 1]
```diff
@@ -649,12 +649,14 @@ class FormCompletionTimeReport(WorkerMonitoringFormReportTableBase, DatespanMixi
...
-        return SingleFormByApplicationFilter.get_value(self.request, self.domain)
+    def selected_form_data(self):
+        data = FormsByApplicationFilter.get_value(self.request, self.domain)
+        if len(data) == 1 and data.values()[0]['xmlns']:
+            return data.values()[0]
 
     @property
     def headers(self):
-        if self.selected_xmlns['xmlns'] is None:
+        if self.selected_form_data:
```
Comment (benrudolph) [inline corehq/apps/reports/standard/monitoring.py]: this seems like the opposite condition? shouldn't it be `if not self.selected_form_data`

File: corehq/apps/reports/standard/monitoring.py [hunk 2]
```diff
@@ -649,12 +649,14 @@ class FormCompletionTimeReport(WorkerMonitoringFormReportTableBase, DatespanMixi
 
     @property
     @memoized
-    def selected_xmlns(self):
-        return SingleFormByApplicationFilter.get_value(self.request, self.domain)
+    def selected_form_data(self):
```
Comment (benrudolph) [inline corehq/apps/reports/standard/monitoring.py]: +1 for the name change

File: corehq/apps/reports/standard/monitoring.py (hunk 1 above)
Comment (snopoke) [inline corehq/apps/reports/standard/monitoring.py]: thanks

Comment (snopoke) [inline corehq/apps/reports/standard/monitoring.py]: fixed here f3016e2

**Comment by benrudolph** [comment]
looks good. no test coverage on that so gonna merge

**Pull request merged**